ansible/playbooks/sap-hana-preconfigure.yaml

```diff
@@ -3,7 +3,14 @@
   remote_user: cloudadmin
   become: true
   become_user: root
+
+  pre_tasks:
+    - name: Source hana install variables
+      include_vars: ./vars/hana_vars.yaml
```

Comment:
This file has `ansible.builtin.include_vars: ./vars/hana_vars.yaml` :green_circle: